deploy/helm/sumologic/values.yaml

```diff
@@ -311,6 +311,9 @@ fluentd:
     output: false
 
   metadata:
+    ## Option to control capturing of annotations by metadata filter plugin.
+    annotation_match:
+      - '"sumologic\\.com.*"'
```

Comment:
I didn't saw this before, but I don't think that adding `"` because helm magically converts list to fluentd format is a good idea. We should rather use some helm template function or write helper to mitigate this. Otherwise we have to explain in values.yaml how to properly build the `fluentd.metadata.annotation_match`